person.rb

```diff
@@ -27,4 +28,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] As I mentioned in the above comment make sure you update the `add_rental()` method so it has ``book``, ``date`` as parameters, and the ``person`` itself should be sent as a parameter to create the new rental achieving the 'has-many' association`. You take a look at **[this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)** for some examples.
e.g.:
```ruby
  def add_rental(book, date)
    Rental.new(book, date, self)
  end
```